tests/unit/components/SharedComponents/PhotoCount/PhotoCount.test.js

```diff
@@ -0,0 +1,33 @@
+import { render, screen } from "@testing-library/react-native";
+import { PhotoCount } from "components/SharedComponents";
+import React from "react";
+
+describe( "PhotosCount", () => {
+  it( "should render correctly", () => {
+    render( <PhotoCount count={10} /> );
+
+    const photoCount = screen.queryByTestId( "photo-count" );
```

Comment:
`queryByTestId` is to be used in situations when you do not expect the element to be present.

If you expect the element to be present, you have to use `getByTestId` (or `findByTestId` if you expect the element to appear after some async changes).

Reply:
Thanks for the explanation !